ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -116,6 +116,13 @@ def main(argv=sys.argv[1:]):
                 'set the AMENT_CPPCHECK_ALLOW_1_88 environment variable to override this.',
                 file=sys.stderr,
             )
+
+            if args.xunit_file:
+                report = {input_file: [] for input_file in files}
+                write_xunit_file(args.xunit_file, report, time.time() - start_time,
```

Comment:
Nitpick: wrap after open parenthesis according to style guide.

Reply:
Fixed.